fontforge/ufo.c

```diff
@@ -1139,9 +1152,11 @@ static int UFOOutputFontInfo(const char *basedir, SplineFont *sf, int layer) {
       // We attempt to get numeric major and minor versions for U. F. O. out of the FontForge version string.
       int versionMajor = -1;
       int versionMinor = -1;
-      if (sf->version != NULL) extractNumericVersion(sf->version, &versionMajor, &versionMinor);
-      if (versionMajor >= 0) PListAddInteger(dictnode,"versionMajor", versionMajor);
-      if (versionMinor >= 0) PListAddInteger(dictnode,"versionMinor", versionMinor);
+      if (sf->version != NULL) {
+          extractNumericVersion(sf->version, &versionMajor, &versionMinor);
+          if (versionMajor >=0) PListAddInteger(dictnode,"versionMajor", versionMajor);
+          if (versionMinor >=0) PListAddInteger(dictnode,"versionMinor", versionMinor);
+      }
```

Comment:
I do not understand what changes here. With the previous code, if sf->version is NULL, then versionMajor and versionMinor do not get written anyway.